### PR TITLE
Flutter home

### PR DESCRIPTION
**Pull request by zolfekar-seleten**

### CODE REVIEW

**Review by hassan4000 — APPROVED** [review]

**Review by hassan4000 — APPROVED** [review]

**Review by hassan4000 — APPROVED** [review]

**Review by hassan4000 — APPROVED** [review]

File: anime-flutter/anime_galaxy/lib/module_anime/repository/anime_details/anime_details.repository.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
...=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1200&q=60 1200w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1296&q=60 1296w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1400&q=60 1400w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1600&q=60 1600w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1800&q=60 1800w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2000&q=60 2000w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2200&q=60 2200w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2400&q=60 2400w, https://images.unsplash.com/photo-1518806118471-f28b20a1d79d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2592&q=60 2592w' ),
```
Comment (mo-kalaleeb) [inline anime-flutter/anime_galaxy/lib/module_anime/repository/anime_details/anime_details.repository.dart]: Isn't this ready in the API?

File: anime-flutter/anime_galaxy/lib/module_anime/ui/widget/anime_details_widget/ainme_details_widget.dart [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        children: [
+          Container(
+            width: MediaQuery.of(context).size.width*0.75,
+            padding: EdgeInsetsDirectional.fromSTEB(5, 5, 5, 10),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.end,
+              children: [
+                Text(
+                  name,
+                  textDirection: TextDirection.rtl,
```
Comment (mo-kalaleeb) [inline anime-flutter/anime_galaxy/lib/module_anime/ui/widget/anime_details_widget/ainme_details_widget.dart]: USe Scaffold Direction. DO NOT USE THIS EVER!

File: anime-flutter/anime_galaxy/lib/module_anime/ui/widget/anime_details_widget/ainme_details_widget.dart [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                        Text(
+                          S.of(context).Share,
+                          textDirection: TextDirection.rtl,
+                          style: TextStyle(
+                            fontSize: 10
+                          ),
+                        ),
+                        IconButton(
+                          icon: Icon(Icons.share),
+                          onPressed: (){},
```
Comment (mo-kalaleeb) [inline anime-flutter/anime_galaxy/lib/module_anime/ui/widget/anime_details_widget/ainme_details_widget.dart:98]: Add TODO Here ;)

File: anime-flutter/anime_galaxy/lib/module_anime/ui/widget/anime_details_widget/ainme_details_widget.dart [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+              ],
+            ),
+          ),
+            Container(
+              width: MediaQuery.of(context).size.width*0.22,
+
+              child: ClipRRect(
+                  borderRadius: BorderRadius.circular(15),
+                  child:  Image(
+                  image: NetworkImage(image),
```
Comment (mo-kalaleeb) [inline anime-flutter/anime_galaxy/lib/module_anime/ui/widget/anime_details_widget/ainme_details_widget.dart]: Please User FadeInImage.assetNetwork to achieve lazy loading here

File: anime-flutter/anime_galaxy/lib/module_auth/ui/screen/auth_screen/auth_screen.dart [hunk 5]
```diff
@@ -129,7 +129,7 @@ class _AuthScreenState extends State<AuthScreen> {
                   Padding(
                     padding: const EdgeInsets.all(16.0),
                     child: Text(
-                      loading == false ? 'Confirm!' : 'Loading!',
+                      loading == false ? S.of(context).confirm : 'Loading!',
```
Comment (mo-kalaleeb) [inline anime-flutter/anime_galaxy/lib/module_auth/ui/screen/auth_screen/auth_screen.dart:132]: Use Intl Translation Package Here Please, not super important but necessary

Comment (mo-kalaleeb) [reply anime-flutter/anime_galaxy/lib/module_auth/ui/screen/auth_screen/auth_screen.dart:132]: These become a pain to find later

File: anime-flutter/anime_galaxy/lib/module_home/ui/widget/points_widget/points_widget.dart [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+            children: [
+              Icon(
+                Icons.star,
+                color: Color(0xfff77f00),
+              ),
+              Container(
+                margin: EdgeInsets.only(top: 10),
+                child: Text(
+                  '$points '+ S.of(context).point,
+                  textDirection: TextDirection.rtl,
```
Comment (mo-kalaleeb) [inline anime-flutter/anime_galaxy/lib/module_home/ui/widget/points_widget/points_widget.dart:31]: Up

**Review by mo-kalaleeb — APPROVED** [review]
Please Check the notes for future release

**Pull request merged**